services.yaml

```diff
@@ -2576,6 +2703,34 @@ definitions:
           type: string
           description: active / pending invitation
           example: active
+  Timer:
+    description: Timer properties
+    type: object
+    properties:
+      id:
+        type: string
+        description: This is an auto generated ID. When creating/editing a timer, the value recevied in this field will be ignored.
+        example: 34
+      class_name:
+        type: string
+        example: DBManager
+      method_name:
+        type: string
+        example: SaveWithRetries
+      threshold:
+        type: integer
+        description: Threshold in ms
+        example: 8000
+      servers:
+        type: array
+        items:
+          type: string
+          example: na-dotCMS-2
+      applications:
```

Comment:
we should support deployments as well

Reply:
at first stage decided to align with the UI which is missing the deployment